rosetta/docs/NATIVE_FP8.md

```diff
@@ -128,7 +128,42 @@ python -m paxml.main \
 
 Please ensure you include the first two flags, `--xla_gpu_enable_reduction_epilogue_fusion=false` and `--xla_gpu_enable_triton_gemm=false`, as they are essential for enabling the FP8 functionality. The additional flags primarily focus on performance enhancement and should also prove beneficial for non-FP8 executions.
 
-### Transformer Engine vs Native FP8 Support
+## Guide for Ninja Users
+
+### Exact pattern that XLA can match for FP8 MatMul
+The specific graph pattern that XLA supports for FP8 matmul is illustrated below:
+
+```
+convert -> multiply -> (x) -> dot
+  bcast ->
+```
+
+XLA will pattern match the above and rewrite it to FP8 matmul when:
+1. `convert`: supports `f8` to [`bf16`|`f16`|`f32`].
+2. `bcast`: broadcasts a [`bf16`|`f16`|`f32`] scalar.
```

Comment:
Does the dtype of 1 and 2 must be the same? Document in all cases if they are or not.